examples/step-69/doc/kind

```diff
@@ -0,0 +1 @@
+time dependent
```

Comment:
please add a tooltip to ``tooltip`` (sorry, comment is at wrong file)

Reply:
You might want to make this a fluid flow problem instead of a time dependent program

tooltip still missing